community/index.tt

```diff
@@ -267,6 +267,9 @@
       <h3>Moderation Team</h3>
       <p>Moderates partitcipation on official community platforms</p>
       <a class="button -primary" href="[%root%]community/teams/moderation.html">Members &amp; Contacts</a>
+      <h3>Documentation Team</h3>
+      <p>Works on improving learning experience and documentation</p>
+      <a class="button -primary" href="[%root%]community/teams/documentation.html">Members &amp; Contacts</a>
```

Comment:
This link is broken: https://nixos.org/community/teams/documentation.html

Reply:
Fixing forward: https://github.com/NixOS/nixos-homepage/pull/885